flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/JsonFunctionsITCase.java

```diff
@@ -313,4 +321,121 @@ private static TestSpec jsonValue() throws Exception {
                                 true,
                                 DataTypes.BOOLEAN().notNull()));
     }
+
+    private static List<TestSpec> jsonQuery() throws Exception {
+        final InputStream jsonResource =
```

Comment:
nit: move this into a helper method for reading a string, other tests use the same lines too?